Mage.Sets/src/mage/cards/a/AllHallowsEve.java

```diff
@@ -96,8 +96,9 @@ public boolean apply(Game game, Ability source) {
                 .map(g -> g.getCards(game))
                 .flatMap(Collection::stream)
                 .filter(card1 -> card1.isCreature(game))
-                .forEach(cards::add);
-        controller.moveCards(card, Zone.BATTLEFIELD, source, game, false, false, true, null);
+                .forEach(allCreatureCardsInGraveyards::add);
+
+        controller.moveCards(allCreatureCardsInGraveyards.getCards(game), Zone.BATTLEFIELD, source, game, false, false, true, null);
```

Comment:
This isn't right. Should be the controller of each card in allCreatureCardsInGraveyards that moves them to the battlefield, not the controller of All Hallow's Eve.

Reply:
Are you going to fix it or should I?  I am looking through the code so that we can add this pull request.

Reply:
Yes. And I plan to also add the tests. Just haven't had a moment to do it